README.md

```diff
@@ -565,6 +565,20 @@ client.waitForTasks(uids: number[], { timeOutMs?: number, intervalMs?: number })
 client.index('myIndex').waitForTasks(uids: number[], { timeOutMs?: number, intervalMs?: number }): Promise<Task[]>
 ```
 
+### Batches <!-- omit in toc -->
+
+#### [Get one batch](https://www.meilisearch.com/docs/reference/api/batches)
+
+```ts
+client.getBatch(uid: number): Promise<Batch>
+```
+
+#### [Get all batches](https://www.meilisearch.com/docs/reference/api/batches)
```

Comment:
Those URLs are sending us to the same place. We probably want to check with documentation and update them ;)

Reply:
The [documentation PR for v1.12](https://github.com/meilisearch/documentation/pull/3068) is not ready yet. 

@guimachiavelli can you share the future URLs with anchor so we can update this? 🙏

Reply:
I haven't started on the batches route, but the URLs will very likely be:

- Multiple batches: https://www.meilisearch.com/docs/reference/api/batchess#get-batches
- Single batch: https://www.meilisearch.com/docs/reference/api/batches#get-one-batch

I should be done with this endpoint on Monday and will let you know if I need to change these URLs for any reason.